src/createServer.js

```diff
@@ -1,3 +1,35 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+
+const { convertToCase } = require('./convertToCase');
+const { parseURL } = require('./parseURL');
+const { validateData } = require('./validateData');
+const { serverResponse } = require('./serverResponse');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const { stringCase, queryString } = parseURL(req.url, req.headers.host);
+    const validation = validateData(queryString, stringCase);
+
+    res.writeHead(200, { 'Content-Type': 'application/json' });
+
+    if (validation.errors.length) {
+      serverResponse(res, 400, validation);
+
+      return;
+    }
+
+    const result = {
+      ...convertToCase(queryString, stringCase),
+      originalText: queryString,
+      targetCase: stringCase,
+    };
+
+    res.end(JSON.stringify(result));
```

Comment:
reuse `serverResponse` here
```suggestion
    serverResponse(res, 200, result);
```